webapp/src/main/webapp/js/individual/individualQtipBubble.js

```diff
@@ -6,48 +6,44 @@ $(document).ready(function(){
     
     $.extend(this, i18nStrings);
 
+    $('head').append('<style id="researchAreaCSS">.qtip { font-size: 14px; max-width: none !important; } .researchAreaTip { background: url(' + imagesPath + '/individual/researchAreaBubble.png) no-repeat; } </style>');
+    padding: '0em',
```

Comment:
Ahh... that line should have been commented out. Just there as a placeholder for migrating the CSS from the previous definition. Thanks! I've removed it.